pages/linux/init.md

```diff
@@ -0,0 +1,23 @@
+# init
+
+> Its primary role is to create processes from a script stored in the file /etc/inittab.
```

Comment:
`primary role`? This implies that it does other things besides the thing you describe here. Perhaps rewording this a bit would help improve clarity? The description should give a short, concise description of what the command does.